Assets/MixedRealityToolkit/InputModule/Scripts/Focus/SimpleSinglePointerSelector.cs

```diff
@@ -1,11 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using MixedRealityToolkit.InputModule.Cursor;
 using MixedRealityToolkit.InputModule.EventData;
 using MixedRealityToolkit.InputModule.Gaze;
 using MixedRealityToolkit.InputModule.InputHandlers;
 using MixedRealityToolkit.InputModule.InputSources;
+using MixedRealityToolkit.UX.Cursors;
```

Comment:
This sucks that now to use simple Gaze and Cursor in my app I need both folders Input and UX?

Reply:
This scenario is needed by pretty much every app. I am supportive of this. Sorry if I should have caught this earlier.

Reply:
Perhaps a UX folder under Input for things that depend purely on Input like Cursor and Motion Controllers?

Reply:
I thought we agreed that all visual representations would be UX?

Also, after my input refactor we won't have the requirements to use a cursor on our pointers.

Reply:
No matter how we approach this, most likely the ux will be dependent on the input folder in some way. But we won't have the Input depending on the ux.

Reply:
When you say "I need to use both UX and Input folders in my project" @NeerajW it makes me shudder.

IMHO, the MRTK project is a full fledged asset, meant to be used / shipped as a whole.  It doesn't bloat the project or cause any issue by leaving the main MRTK folder intact.  in some way, that would be wholly resolved by having all the code wrapped in a single DLL (but let's not go there)

With Unity, there isn't any practical reason to remove asset code from a project and if people feel they need to , then we need to address that.  Simply put, just removing "stuff" without understanding why people do it is likely to cause runtime issues later because they removed too much.

Reply:
@StephenHodgson I don't recall a discussion around us agreeing on everything UX but I could be wrong for sure.

@DDReaper what you said is the exact opposite feedback I get. People don't want to import a ton of code into their projects. Hence I did the semi-modularization by reducing cross feature dependencies.

My scenario is fairly simple as I stated above: In my app I need Input from gaze and I need to visualize it. Looks like I will need a lot of code to do that contained in two separate feature folders Input and UX. Previously I could just import Input and be on my way.
These type of massive code pieces are a part of what deter people from using and contributing.

I agree from an asset prespective having things in UX makes sense. I just want to make sure we address the feedback on MRTK around poor modularization. :(

Reply:
May be we solve this with Unity packages for scenarios. @keveleigh thoughts? We chatted on this yesterday briefly.

Reply:
Based on this @NeerajW , then we may need a small re-think on the Multi-VR solution to breed more modularization, if we intend (rather large) chunks to be able to be split off.

Reply:
@DDReaper I think we should def focus on making some feature areas decoupled. For example spatial mapping and understanding shouldn't have dependencies on input for example.